pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -62,6 +64,32 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		if err != nil {
 			return nil, err
 		}
+
```

Comment:
For clarification. when you say outside of the create block, are you talking about moving the proxy protocol logic entirely out of the `ensureLoadBalancer` method (like with `updateInstanceSecurityGroupsForLoadBalancer`) or simply talking about refactoring the duplicated the code out into a separate method?